src/ageml/commands.py

```diff
@@ -164,12 +164,28 @@ def configure_args(self, args):
         else:
             args.model_params = {}
 
-        # Set hyperparameter grid search value
-        if len(args.hyperparameter_tuning) > 1 or not args.hyperparameter_tuning[0].isdigit():
+        # Parse hyperparameter_tuning values
+        hyperparam_tuning = args.hyperparameter_tuning
+        if not hyperparam_tuning[0].isdigit():
             raise ValueError("Hyperparameter grid points must be a non negative integer.")
         else:
-            args.hyperparameter_tuning = args.hyperparameter_tuning[0]
-            args.hyperparameter_tuning = int(convert(args.hyperparameter_tuning))
+            args.hyperparameter_tuning = int(convert(hyperparam_tuning[0]))
+
+        hyperparameter_params = {}
+        if len(hyperparam_tuning) > 1:
+            for item in hyperparam_tuning[1:]:
+                if item.count("=") != 1:
+                    err_msg = (
+                        "Hyperparameter tuning parameters must be in the format "
+                        "param1=value1_low,value1_high param2=value2_low, value2_high..."
+                    )
```

Comment:
should also check that always two values given a low and a high. What happens if someone gives C=1,2,3? this should through an error. As the user should write C=1,3 and ht=3 to have 3 hyperparameter points 1,2,3. Also ht should be a minimum of 2.

Reply:
What about hyperparameters like kernels? where you want to choose different kernels. This should not be affected by ht.